src/site/static/eligibility.liquid

```diff
@@ -552,61 +766,110 @@ pageClass: "page-public-assistance"
-      // regardless of form input.  For conditional next-page selection, 
-      // add the logic to customPageLinking().
+
+// By default, each page will advance to the next page in the sequence,
+// regardless of form input.  For conditional next-page selection, 
+// add the logic to customPageLinking().
       pages[j].next = function() {
         return pages[j + 1];
       };
-      // Store a hash map of page id to the page itself for faster page
-      // retrieval when defining custom page linking.
+
+// Store a hash map of page id to the page itself for faster page
+// retrieval when defining custom page linking.
       pageById[pages[j].id] = pages[j];
     }
     customPageLinking(pageById);
   }
 
-  // Holder for all page advancing logic that is more complex than simply
-  // "the next page in the sequence".
+// Holder for all page advancing logic that is more complex than simply
+// "the next page in the sequence".
   function customPageLinking(pageById) {
-    // In this example, the page 'page-select-likes' advances to
-    // the page 'page-today' only if a certain form checkbox is selected.
-    // Otherwise, the page 'page-select-likes' advances to the page
-    // 'page-color'.
-    pageById["page-select-likes"].next = function() {
-      const likesDays = document.getElementById("likes-days").checked;
-      if (likesDays) {
-        return pageById["page-today"];
+
+// In this example, the page 'page-select-likes' advances to
+// the page 'page-today' only if a certain form checkbox is selected.
+// Otherwise, the page 'page-select-likes' advances to the page
+// 'page-color'.
+    pageById["page-yourself-start"].next = function() {
+
+      const isUnderEighteen = document.getElementById("age").value < 18 && document.getElementById("age").value > 0
+      const isDisabled = document.getElementById("disabled").checked;
+      const isBlind = document.getElementById("blind").checked;
+      const isDeaf = document.getElementById("deaf").checked;
+      const isVeteran = document.getElementById("veteran").checked;
+      const isPregnant = document.getElementById("pregnant").checked;
+      const isFeeding = document.getElementById("feeding").checked;
+      const isCitizen = document.getElementById("citizen").checked;
+
+      if (isUnderEighteen) {
+        return pageById["page-head-of-household"];
+      }
+      if (isDisabled || isBlind || isDeaf) {
+        return pageById["page-disability-details"];
+      }
+      if (isVeteran) {
+        return pageById["page-veteran-details"];
+      }
+      if (isPregnant || isFeeding) {
+        return pageById["page-household-start"];
+      }
+      if (! isCitizen) {
+        return pageById["page-immigration-status"];
+      } else {
+        return pageById["page-household-start"];
+      }
+    };
+
+    pageById["page-head-of-household"].next = function() {
+      const isDisabled = document.getElementById("disabled").checked;
```

Comment:
No need to change it in this PR since you will be doing a lot of work on this code for the next PR, but I figured out a nice way to handle these next() functions for the [income pages](https://github.com/philhawksworth/the-united-effort-orginization/blob/b4df808a27cf696cd0c714e60225737d11f1738e/src/site/static/eligibility.liquid#L638).

I think you can use a similar pattern here.  The idea is that you check whether the next page in sequence should be shown or not.  If it should be shown, you return that page.  Otherwise, you call that page's next() function.  This means you don't have to duplicate a bunch of code.  Imagine a series of 3 pages in a section: page1, page2, and page3. After page3 the next section begins.

```
pageById["page1"].next = function () {
  if (showPage2()) {
    return pageById["page2"];
  }
  return pageById["page2"].next();
}

pageById["page2"].next = function () {
  if (showPage3()) {
    return pageById["page3"];
  }
  return pageById["page3"].next();
}
```

compare that to the following, which should have the same behavior:
```
pageById["page1"].next = function () {
  if (showPage2()) {
    return pageById["page2"];
  } else if (showPage3()) {
    return pageById["page3"];
  }
  return pageById["next-section-page1"];
}

pageById["page2"].next = function () {
  if (showPage3()) {
    return pageById["page3"];
  }
  return pageById["next-section-page1"];
}
```

As the number of pages in a section grows, the second solution will get more and more duplicated code because each `.next()` definition needs to re-define all the possible paths.